keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -385,6 +476,8 @@ Other components:
    update to running Pods.
 1. Allow resizing local ephemeral storage.
 1. Allow resource limits to be updated (VPA feature).
+1. Kubelet retry resizes (deferred)
```

Comment:
We have retries today, so we don't need this point.

Reply:
ACK